htdocs/webportal/public/includes/jquery/css/base/theme.css

```diff
@@ -0,0 +1,443 @@
+/*!
```

Comment:
Jquery is already embeded into dolibarr, you must reuse the existing one.
Having the lib js outside public is not a problem, static js files are already allowed for public access by adding fail2ban or http basic rules that exclude them for filtering when called out of the backoffice (for example with a Files rule inside the vietual host as defined into the dev/setup/apache/vietualhost example).
So you can consider the files as if they are public even if not inside the public dir (not a danger, they are all static files)

Reply:
It has been done